docs/tutorials/control/air-quality-fleet.md

```diff
@@ -0,0 +1,827 @@
+   Follow the [instructions to add a fragment mod](/fleet/configure-a-fleet/#use-fragment_mods) to your machine's config, using the following JSON template:
+
+   ```json {class="line-numbers linkable-line-numbers"}
+   "fragment_mods": [
+   {
+     "fragment_id": "<REPLACE WITH YOUR FRAGMENT ID>",
+     "mods": [
+       {
+         "$set": {
+           "components.PM_sensor.attributes.usb_interface": "<REPLACE WITH THE PATH TO THE SENSOR ON YOUR MACHINE>"
+         }
+       }
+     ]
+   }
+   ],
+   ```
+
+   Replace the values with your fragment ID and with the USB path you identify.
+   If you named your sensor something other than `PM_sensor`, change the sensor name in the template above.
+
+3. Repeat this process for each machine that needs a different `usb_interface` value.
+   If you have lots of machines with one `usb_interface` value, and lots of machines with a second one, you might consider duplicating the fragment, editing that value, and using that second fragment instead of the first one for the applicable machines, rather than using a fragment mod for each of the machines.
+   You have options.
+
+   {{% /expand%}}
+
+## Test data sync
+
+Next, check that data is being synced from your sensors to the cloud:
+
+1.  Open your [**DATA** page](https://app.viam.com/data).
+2.  Click the **Sensors** tab within the data page.
+3.  If you have sensor data coming from machines unrelated to this project, use the filters on the left side of the page to view data from only your air quality sensors.
+    Click the **Tags** dropdown and select the `air-quality` tag you applied to your data.
+    You can also use these filters to show the data from one of your air quality sensors at a time by typing a machine name into the **Machine name** box and clicking **Apply** in the lower-left corner.
+
+    {{<imgproc src="/tutorials/air-quality-fleet/synced-data.png" resize="x1100" declaredimensions=true alt="The sensor readings that have synced to the DATA page." style="max-width:600px" >}}
+
+Once you've confirmed that data is being collected and synced correctly, you're ready to start building a dashboard to display the data.
+If you'd like to graph your data using a Grafana dashboard, try our [Visualize Data with Grafana tutorial](/tutorials/services/visualize-data-grafana/).
+If you'd like to create your own customizable dashboard using the Viam TypeScript, continue with this tutorial.
+
+## Code your custom TypeScript dashboard
+
+The [Viam TypeScript SDK](https://ts.viam.dev/) allows you to build custom web interfaces to interact with your machines.
+For this project, you'll use it to build a page that displays your air quality sensor data.
+You'll host the website locally on your Linux or MacOS computer, and view the interface in a web browser on that computer.
```

Comment:
I think the typescript SDK should be supported on all platforms? Do you know it's not?

Reply:
Ah you're right!